### PR TITLE
Add Styling to the Notes App 

### PR DESCRIPTION
**Pull request by annawinther**
## Description
- Bootstrap styling with Styled-Components
- Added React-Router
- Sign out button using amplify Auth
- Sign up form using amplify react UI components
- Added general styling to the app 
- Added pagination using the nextToken

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -27,11 +33,25 @@ const App = ({
     fetchNotesAction();
   }, [fetchNotesAction]);
 
-  const onCreateNote = async () => {
+  const onCancel = () => {
+    setFormState(initialState);
+    setEdit(false);
+    history.push('/');
```
Comment (dibell) [inline src/App.js:39]: Why all the push('/')?

File: src/App.js [hunk 2]
```diff
@@ -57,36 +78,54 @@ const App = ({
...
-        loading={isLoading}
-        errors={errors}
-        deleteNote={onDeleteNote}
-        updateNote={onUpdateNote}
-      />
-    </div>
+    <AppContainerStyled>
+      <Naviagtion />
+      <Switch>
+        <Route path="/form">
```
Comment (dibell) [inline src/App.js:88]: Perhaps a route that includes the note ID would be useful (for bookmarking etc).
e.g. /note/<uuid>

File: src/components/Note.js [hunk 3]
```diff
@@ -1,27 +1,66 @@
...
+const Note = ({
+  note, deleteNote, updateNote,
+}) => {
   const { name, description } = note;
+  const [open, setOpen] = useState(false);
+  const [rotate, setRotate] = useState(false);
+
+  const toggleOpen = () => {
+    if (open) {
+      setOpen(false);
```
Comment (dibell) [inline src/components/Note.js:15]: setOpen(!open) maybe?

File: src/components/Note.js [hunk 4]
```diff
@@ -1,27 +1,66 @@
...
+    } else {
+      setOpen(true);
+    }
+  };
+
+  const rotateClick = () => {
+    if (rotate) {
+      setRotate(false);
```
Comment (dibell) [inline src/components/Note.js:23]: also here

File: src/components/Note.js [hunk 5]
```diff
@@ -1,27 +1,66 @@
...
-          onClick={() => updateNote(note)}
-          color="black"
-          size={22}
-        />
-      </div>
-    </div>
+    <ListItems className="list-group">
+      <li
+        className="list-group-item border-0 d-flex justify-content-between align-items-center"
+        onClick={() => { toggleOpen(); rotateClick(); }}
```
Comment (dibell) [inline src/components/Note.js:33]: rotate?

File: src/App.js (hunk 2 above)
Comment (annawinther) [reply src/App.js:85]: This is a good idea! I'll work on that later, if I have more time :)

**Review by dibell — APPROVED** [review]

**Pull request merged**